Common/Servers/SolarSystem/src/SolarSystemBodyImpl.cpp

```diff
@@ -395,7 +397,8 @@ void SolarSystemBodyImpl::BodyPosition(TIMEVALUE &time)
 
         m_sitex -> setTime(time_utc) ;  
         m_body_xephem->compute( m_sitex );
-  //      m_body_xephem->report();
+		  m_sitex ->stampa();        
+        m_body_xephem->report();
```

Comment:
Do these last 2 lines print anything for debugging purposes in production? If so, we should comment them out. If not just leave them here.